viewer/src/datamodels/cad/styling/ByNodePropertyNodeSet.ts

```diff
@@ -35,9 +37,10 @@ export class ByNodePropertyNodeSet extends NodeSet {
   private readonly _revisionId: number;
   private readonly _options: Required<ByNodePropertyNodeSetOptions>;
   private _fetchResultHelper: PopulateIndexSetFromPagedResponseHelper<Node3D> | undefined;
+  private _filter = {};
 
   constructor(client: CogniteClient, model: Cognite3DModel, options: ByNodePropertyNodeSetOptions = {}) {
-    super();
+    super(ByNodePropertyNodeSet.name);
```

Comment:
This _might_ be a bit dodgy as it might be that `ByNodePropertyNodeSet` is renamed in minified code, so we should probably stick to using a constant https://stackoverflow.com/questions/13613524/get-an-objects-class-name-at-runtime

```suggestion
    super('ByNodePropertyNodeSet');
```